lib/utils/version.js

```diff
@@ -115,11 +115,14 @@ const utils = {
             return null;
         }
 
-        if (activeMajor === 1 && activeVersion !== latestV1 && semver.diff(activeVersion, versionToUse) === 'major') {
+        const latestMajor = latestMajorVersions[`v${activeMajor}`];
+
+        if (activeVersion !== latestMajor && semver.diff(activeVersion, versionToUse) === 'major') {
             const majorToUse = semver.major(versionToUse);
+
             throw new CliError({
-                message: `You are trying to update to Ghost v${majorToUse}, but your blog is not on the latest Ghost 1.0 version`,
-                help: 'Instead run "ghost update v1".'
+                message: `You are trying to update to Ghost v${majorToUse}, but your blog is not on the latest Ghost ${activeMajor}.0 version`,
```

Comment:
Would this be a more correct message?

```suggestion
                message: `You are trying to update to Ghost v${majorToUse}, but your blog is not on the latest Ghost v${activeMajor} version`,
```

Reply:
yup, would definitely be more consistent - I'll fix

Reply:
or maybe the second part might be better worded "but your blog is not running the latest Ghost v{activeMajor}"